manual/wasm_overview.wiki

```diff
@@ -0,0 +1,56 @@
+= Wasm_of_ocaml =
+
+== Overview ==
+
+Wasm_of_ocaml is a compiler from OCaml bytecode programs to WebAssembly.
+It provides an alternative way to run pure OCaml programs in JavaScript environments like browsers and Node.js.
+
+The compiler is provided by the wasm_of_ocaml-package. The <<a_manual chapter="overview" |Js_of_ocaml libraries>> are compatible with this compiler.
+
+== Installation
+
+The easiest way to install wasm_of_ocaml is to use opam.
+{{{opam install wasm_of_ocaml-compiler js_of_ocaml js_of_ocaml-ppx js_of_ocaml-lwt}}}
+
+== Usage ==
+
+  Your program must first be compiled using the OCaml bytecode
+  compiler {{{ocamlc}}}.  JavaScript bindings are provided by the
+  {{{js_of_ocaml}}} package and the syntax extension by the
+  {{{js_of_ocaml-ppx}}} package
+{{{
+      ocamlfind ocamlc -package js_of_ocaml -package js_of_ocaml-ppx \
+          -linkpkg -o cubes.byte cubes.ml
+}}}
+  Then, run the {{{wasm_of_ocaml}}} compiler to produce Wasm code:
+{{{
+      wasm_of_ocaml cubes.byte
+}}}
+
+This produces a Javascript loading script {{{cube.js}} and a directory
+{{{cube.assets}} containing the Wasm code.
+
+=== with dune ===
+  Dune has native support for wasm_of_ocaml.
```

Comment:
Should we mentioned the dune version or the dune lang required to enable the wasm support?